### ISSUE TITLE
Locale-agnostic datetime

### ISSUE THREAD

**Issue by TicClick** [ppy/osu-wiki#9987]
something we run into occasionally is that different locales have different date time formats: [[1]](https://github.com/ppy/osu-wiki/pull/9483) [[2]](https://github.com/ppy/osu-wiki/pull/7360) [[3]](https://github.com/ppy/osu-wiki/pull/9986) [[4]](https://github.com/ppy/osu-wiki/issues/6509)

peppy has suggested a few times that date formatting should be done by the website, while the wiki will have datetimes in the same format, such as [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) `[2022-12-31T12:34:56Z]`

<details>
<summary>2017-07-13 @ #osu-wiki</summary>

https://discord.com/channels/188630481301012481/218677502141399041/334857793284734978

> 
> [2:45 AM] peppy: honestly we should have a custom datetime formatter
> [2:45 AM] peppy: ie the markdown version should be an rfc datetimestamp 
> [2:46 AM] MegaApple_Pi: hmm, so should we leave it as-is for the time being? 
> [2:47 AM] peppy: for dates?
> [2:47 AM] MegaApple_Pi: yes
> [2:47 AM] peppy: basically leave it "as is" as long as everything is using the same standard
> [2:47 AM] peppy: and it's greppable and regexable
> [2:47 AM] peppy: we don't want to have to manually go through and change each instance when we modify the underlying system
> [2:57 AM] kj415j45: ehh, "standard" actually some times just suit 1 language
> [2:57 AM] kj415j45: like data format
> [2:58 AM] kj415j45: english use DD. MMMM YYYY
> [2:58 AM] peppy: as long as it's standard to the whole wiki
> [2:59 AM] peppy: we can make it more standard using a regex
> [2:59 AM] kj415j45: when chinese YYYY年MM月DD日
> [2:59 AM] peppy: i mean it should be 1985-04-12T23:20:50.52Z in the final representation
> [2:59 AM] peppy: [1985-04-12T23:20:50.52Z] or something
> [3:00 AM] peppy: and the stie should use html5 date and custom tooltips to display, like it does everywhere else
> [3:00 AM] peppy: also apply local timezones
> [3:00 AM] peppy: https://cdn.discordapp.com/attachments/218677502141399041/334861602547433474/unknown.png
> [3:00 AM] peppy: this thing you see everywhere

</details>

thoughts? this will require an osu-web issue

**Comment by Walavouchey** [comment]
yes

**Comment by TicClick** [comment]
my own suggestion is use DDDD-MM-YY anyway when we're not interested in exact time, and DDDD-MM-YYThh:mm:ssZ when we are

**Comment by Walavouchey** [comment]
closing after [discussion on github](https://discord.com/channels/188630481301012481/218677502141399041/1146486812370546851). the underlying issue is mostly tables in translations not having their date formats converted, which can be fixed by automating the copy process

**Comment by cl8n** [comment]
that or just generally be more careful when knowingly overstepping what you're capable of editing. non-native edits on translations should always seek native approval if it involves the language at all imo, and it's also not the end of the world to tack on an `outdated_since` even if the edit to EN was small.

**Comment by DoryanR** [comment]
> that or just generally be more careful when knowingly overstepping what you're capable of editing. non-native edits on translations should always seek native approval if it involves the language at all imo

I tend to agree with this point. It's not the first time I've picked up errors when editing EN files. And I find it a waste of time to create a PR to correct these errors.

> and it's also not the end of the world to tack on an `outdated_since` even if the edit to EN was small.

Frankly, if it's just a modification of dates or links, it's a waste of time for us to create a PR that has to be reviewed by a native (like #9992), when it can be easily modified during the modification PR.
As long as it doesn't affect paragraph changes or sentence meaning, I don't think it's necessary to outdate the translations.

The idea I had at the beginning in #9986 was to create a CI for translations that have a `DD/MM/YYYY` date format. Honestly, I didn't really understand why it would be necessary to modify the date format for EN pages, whereas CIs for translations would have been "simpler" to set up.

Personally, I can't read `YYYY/MM/DD' dates. With experience, I'm getting the hang of it, but it still requires a considerable effort, and I think that those who aren't used to reading dates will look elsewhere. That's why we try to change them in our FR translations. To make it easier and more convenient to read.
